packages/create-expo/babel.config.js

```diff
@@ -1,11 +1,2 @@
-module.exports = function (api) {
-  api.cache(true);
-  return {
-    // Only use this when running tests
-    env: {
-      test: {
```

Comment:
Is this no longer test-only?

Reply:
We can actually drop the whole babel config.
- [NCC will always build with `typescript`, not babel](https://github.com/vercel/ncc/blob/0.38.0/src/index.js#L356-L371)
- Jest should be testing with `ts-jest`, not babel